README.md

```diff
@@ -5,6 +5,18 @@ JavaScript libraries to:
 * Create and send private transactions 
 * Create, delete, and find privacy groups
 
+## ⚠️ Project Deprecation Notice ⚠️
+
+web3js-eea library will be deprecated on December 31st 2021 date from which we will stop supporting the project.
```

Comment:
non-blocking: to avoid duplicating `deprecation`, we can say here `We will stop supporting the project from December 31st 2021` and the next paragraph explains the deprecation part. does it makes sense?

Reply:
It makes it clearer for users. I prefer to keep it this way. Thanks!